doc/overview/faq.rst

```diff
@@ -383,8 +383,8 @@ order of difficulty):
    :ref:`mne watershed_bem`.
 2. Changing the ``--atlas`` and ``--gcaatlas`` options of
    :ref:`mne watershed_bem`.
-3. Manually editing the meshes (see `this tutorial
-   <https://github.com/ezemikulan/blender_freesurfer>`__.
+3. Manually editing the meshes (see :ref:`this tutorial
+   <Editing BEM surfaces in Blender>`).
```

Comment:
/home/circleci/project/doc/overview/faq.rst:386: WARNING: undefined label: editing bem surfaces in blender (if the link has no caption the label must precede a section header)
g